tests/Aspire.Hosting.Redis.Tests/RedisFunctionalTests.cs

```diff
@@ -566,6 +566,21 @@ public async Task RedisInsightWithDataShouldPersistStateBetweenUsages(bool useVo
             else
             {
                 bindMountPath = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName());
+
+                if (!Directory.Exists(bindMountPath))
+                {
+                    Directory.CreateDirectory(bindMountPath);
+                }
+
+                if (!OperatingSystem.IsWindows())
+                {
+                    var permissions = UnixFileMode.UserRead | UnixFileMode.UserWrite | UnixFileMode.UserExecute |
+                                      UnixFileMode.GroupRead | UnixFileMode.GroupWrite | UnixFileMode.GroupExecute |
+                                      UnixFileMode.OtherRead | UnixFileMode.OtherWrite | UnixFileMode.OtherExecute;
+
+                    File.SetUnixFileMode(bindMountPath, permissions);
```

Comment:
We shouldn't be doing this in the test - this seems like a product bug.

Reply:
It's something, but I don't know why it happens on this gha run.